doc/howto/storage_configure.md

```diff
@@ -34,12 +34,12 @@ You can also edit the storage volume configuration by using the following comman
 
     lxc storage volume edit <pool_name> <volume_name>
 
-## Configure default values for storage volumes
+## Configure default values for custom storage volumes
```

Comment:
These settings also apply to non-custom volumes (when not overridden via the instance's config).